test/controllers/columns_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class ColumnsControllerTest < ActionDispatch::IntegrationTest
+  test "should get new" do
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->